boltons/classutils.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+"""Python's built-in :mod:`functools` module builds several useful
+utilities on top of Python's first-class function support.
+``classutils`` attempts to do the same for metaprogramming with
+classes, types and instances.
+"""
+
+import weakref
+
+
+def gencls(*bases, **extras):
```

Comment:
I'm skeptical that this one-liner is a common enough trope among python programmers to warrant inclusion. It's just such a thin wrapper around type that I'd expect the standard code review response would be explicit better than implicit: spell out the class.